app/components/primer/beta/border_box.pcss

```diff
@@ -89,7 +89,7 @@
   border-bottom: var(--borderWidth-thin) solid var(--borderColor-default);
 
   /* Ensures bottom-border doesn't poke out when .Box-body used without box-footer */
-  &:last-of-type {
+  &:last-child {
```

Comment:
An issue with this selector is we often have hidden elements at the end of the rows in our markup

Reply:
I tried to re-create such an example, but failed. When looking at the html of the BorderBox, there is no hidden element defined, and the slot definitions do not accept blocks as arguments either.

```
<%= render Primer::BaseComponent.new(**@system_arguments) do %>
  <%= header %>
  <%= body %>
  <% if rows.any? %>
    <%= render Primer::BaseComponent.new(**@list_arguments) do %>
      <% rows.each do |row| %>
        <%= row %>
      <% end %>
    <% end %>
  <% end %>
  <%= footer %>
<% end %>
```

Could you give me a hint, on how to create an example with hidden elements at the end of the rows?